File: VB/MainViewModel.vb
Imports DevExpress.Core
Imports DevExpress.Mvvm

Public Class MainViewModel
    Inherits ViewModelBase

    Public Sub New()
        Me.ColorSchemeName = Me.ColorSchemeNames(0)
    End Sub

    Protected _ColorSchemeNames As List(Of String)
    Public ReadOnly Property ColorSchemeNames() As List(Of String)
        Get
            If Me._ColorSchemeNames Is Nothing Then
                Me._ColorSchemeNames = New List(Of String)(New String() {ThemeManager.GenericSchemeName, ThemeManager.Win8SchemeName})
            End If
            Return Me._ColorSchemeNames
        End Get
    End Property

    Public Property ColorSchemeName() As String
        Get
            Return ThemeManager.CurrentColorSchemeName
        End Get
        Set(ByVal value As String)
            ThemeManager.CurrentColorSchemeName = value
            Me.RaisePropertyChanged("ColorSchemeName")
        End Set
    End Property


End Class

File: CS/MainViewModel.cs
using DevExpress.Core;
using DevExpress.Mvvm;
using System.Collections.Generic;

namespace App68 {
    public class MainViewModel : ViewModelBase {

        public MainViewModel() {
            this.ColorSchemeName = this.ColorSchemeNames[0];
        }

        protected List<string> _ColorSchemeNames;
        public List<string> ColorSchemeNames {
            get {
                if (this._ColorSchemeNames == null) {
                    this._ColorSchemeNames = new List<string>() { ThemeManager.GenericSchemeName, ThemeManager.Win8SchemeName };
                }
                return this._ColorSchemeNames;
            }
        }

        public string ColorSchemeName {
            get {
                return ThemeManager.CurrentColorSchemeName;
            }
            set {
                ThemeManager.CurrentColorSchemeName = value;
                this.RaisePropertyChanged("ColorSchemeName");
            }
        }


    }
}
